components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -2264,9 +2264,11 @@ private void translateExperiment(Element root) throws FormatException {
         Element group = (Element) groups.item(i);
 
         Element position = getFirstNode(group, "Position");
-        if (position != null) {
-          int tilesX = Integer.parseInt(getFirstNodeValue(group, "TilesX"));
-          int tilesY = Integer.parseInt(getFirstNodeValue(group, "TilesY"));
+        String tilesXValue = getFirstNodeValue(group, "TilesX");
+        String tilesYValue = getFirstNodeValue(group, "TilesY");
```

Comment:
Is it worth making use of the data parsing API introduced in https://github.com/openmicroscopy/bioformats/pull/2285 to catch all cases of `NumberFormatException`?